### ISSUE TITLE
Failed to activate the tidy-markdown package

### ISSUE THREAD

**Issue by joshhaug** [slang800/atom-tidy-markdown#9]
After updating Atom I got a fatal error. 

**Atom Version**: 0.196.0 :warning: **in 1.0 API Preview Mode** :warning:
**System**: Mac OS X 10.10.2
**Thrown From**: [tidy-markdown](https://github.com/slang800/atom-tidy-markdown) package, v0.2.2
### Stack Trace

Failed to activate the tidy-markdown package

```
At undefined is not a function

TypeError: undefined is not a function
    at new TidyMarkdown (/Users/joshhaug/.atom/packages/tidy-markdown/lib/tidy-markdown.coffee:12:37)
    at Object.module.exports.activate (/Users/joshhaug/.atom/packages/tidy-markdown/lib/index.coffee:12:34)
    at Package.module.exports.Package.activateNow (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:222:19)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/package.js:203:30
    at Package.module.exports.Package.measure (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:147:15)
    at Package.module.exports.Package.activate (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:195:14)
    at PackageManager.module.exports.PackageManager.activatePackage (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:434:21)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:418:29
    at Config.module.exports.Config.transact (/Applications/Atom.app/Contents/Resources/app.asar/src/config.js:311:16)
    at PackageManager.module.exports.PackageManager.activatePackages (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:413:19)
```
### Config

``` json
{
  "core": {
    "themes": [
      "one-light-ui",
      "one-dark-syntax"
    ],
    "disabledPackages": [
      "markdown-preview-plus"
    ]
  }
}
```
### Installed Packages

``` coffee
# User
language-swift, v0.4.0
markdown-pdf, v1.3.5
minimap, v4.8.0
tidy-markdown, v0.2.2

# Dev
No dev packages
```

**Comment by notslang** [comment]
ah, crap - I'm still using a bunch of deprecated functions ATM. thanks for the report!

**Comment by notslang** [comment]
fixed with afcbcc919846d9db8037d4e19cdf39d129cbb70f

**Comment by braamvandyk** [comment]
This seems to re-occurring:

Failed to activate the tidy-markdown package
Cannot find module 'parse5'
Hide Stack Trace
Error: Cannot find module 'parse5'
    at Module._resolveFilename (module.js:543:15)
    at Module._resolveFilename (C:\Users\vdykb\AppData\Local\atom\app-1.30.0\resources\electron.asar\common\reset-search-paths.js:35:12)
    at Function.get_Module._resolveFilename (C:\Users\vdykb\AppData\Local\atom\app-1.30.0\resources\app\src\module-cache.js:354:58)
    at Module.require (file:///C:/Users/vdykb/AppData/Local/atom/app-1.30.0/resources/app.asar/static/index.js:40:43)
    at require (C:\Users\vdykb\AppData\Local\atom\app-1.30.0\resources\app\src\native-compile-cache.js:66:33)
    at Object.<anonymous> (file:///C:/Users/vdykb/.atom/packages/tidy-markdown/node_modules/tidy-markdown/lib/index.coffee:5:19)
    at Object.<anonymous> (file:///C:/Users/vdykb/.atom/packages/tidy-markdown/node_modules/tidy-markdown/lib/index.coffee:1:1)
    at Object.<anonymous> (file:///C:/Users/vdykb/.atom/packages/tidy-markdown/node_modules/tidy-markdown/lib/index.coffee:1:1)
    at Module.get_Module._compile (C:\Users\vdykb\AppData\Local\atom\app-1.30.0\resources\app\src\native-compile-cache.js:106:36)
    at Object.value [as .coffee] (C:\Users\vdykb\AppData\Local\atom\app-1.30.0\resources\app\src\compile-cache.js:240:29)
    at Module.load (module.js:561:32)
    at tryModuleLoad (module.js:504:12)
    at Function.Module._load (module.js:496:3)
    at Module.require (file:///C:/Users/vdykb/AppData/Local/atom/app-1.30.0/resources/app.asar/static/index.js:47:45)
    at require (C:\Users\vdykb\AppData\Local\atom\app-1.30.0\resources\app\src\native-compile-cache.js:66:33)
    at Object.<anonymous> (C:\Users\vdykb\.atom\packages\tidy-markdown\lib\index.coffee:6:20)
    at Object.<anonymous> (C:\Users\vdykb\.atom\packages\tidy-markdown\lib\index.coffee:87:4)
    at Object.<anonymous> (C:\Users\vdykb\.atom\packages\tidy-markdown\lib\index.coffee:91:3)
    at Module.get_Module._compile (C:\Users\vdykb\AppData\Local\atom\app-1.30.0\resources\app\src\native-compile-cache.js:106:36)
    at Object.value [as .coffee] (C:\Users\vdykb\AppData\Local\atom\app-1.30.0\resources\app\src\compile-cache.js:240:29)
    at Module.load (module.js:561:32)
    at tryModuleLoad (module.js:504:12)
    at Function.Module._load (module.js:496:3)
